bigchaindb/log.py

```diff
@@ -3,10 +3,10 @@
 
 from bigchaindb.common.exceptions import ConfigurationError
 from logging.config import dictConfig as set_logging_config
-from os.path import expanduser, join
+import os
 
 
-DEFAULT_LOG_DIR = expanduser('~')
+DEFAULT_LOG_DIR = os.getcwd()
```

Comment:
It depends on where you run the command from, not on how and where you install it.

Reply:
It will be derived from the directory where you are running `bigchaindb start` from and in Docker we set the WORKDIR to `/usr/src/app`, the logs will be generated there.

Reply:
I would be good to make a note and mention such details in the doc, in this PR or a future PR whenever the related docs are added.

Reply:
I will note down where the logs are stored in the Monit setup section.